pkgs/desktops/gnome/extensions/caffeine/default.nix

```diff
@@ -2,13 +2,13 @@
 
```

Comment:
Please remove the gnome from inputs.

Reply:
I've also changed the license mentioned by the bot.